src/cli/cli.cpp

```diff
@@ -1898,6 +1898,9 @@ void Interpreter::ProcessLine(char *aBuf, uint16_t aBufLength, Server &aServer)
         {
             argv[argc++] = cmd;
         }
+
+        VerifyOrExit(argc < kMaxArgs,
+                     sServer->OutputFormat("Error %d, Max %d\r\n", kThreadError_TooManyArgs, kMaxArgs));
```

Comment:
Rather than defining an error code here, we should just print the error as it is.  For example:

`sServer->OutputFormat("Error: too many args (max = %d)\r\n", kMaxArgs);`